wallet/README.md

```diff
@@ -0,0 +1,31 @@
+wallet
+========
+
+[![Build Status](https://travis-ci.org/btcsuite/btcwallet.png?branch=master)]
+(https://travis-ci.org/btcsuite/btcwallet)
+
+## Feature Overview
+
+

```

Comment:
The main feature of this package is the integration of all the other wallet components (waddrmgr, txstore, and chain) and the Wallet type is 'runnable', so it will be continuously updating itself against changes notified by the remote btcd instance.